docs/topics/cli-run.adoc

```diff
@@ -63,21 +67,36 @@ The following command analyzes the `com.acme` and `org.apache` packages of the l
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/{mta-cli} --input /path/to/jee-example-app-1.0.0.ear \
+
+$ <{ProductShortName}_HOME>/bin/windup-cli --input /path/to/jee-example-app-1.0.0.ear \
+
```

Comment:
The `bin` folder should be removed.

Should we reference `windup-cli` directly instead using a placeholder `{mta-cli}` like we were doing?